_posts/2021-11-08-hhvm-4.136.markdown

```diff
@@ -0,0 +1,28 @@
+---
+title: "HHVM 4.136"
+layout: post
+author: atry
+category: blog
+---
+
+HHVM 4.136 is released! HHVM 4.131&ndash;4.135 remain supported, as do the 4.102 and 4.128 LTS releases.
+
+# Highlights
+
+- Add IDE quickfix for missing interface methods.
+- [The `readonly` feature](https://docs.hhvm.com/hack/readonly/introduction) is no longer considered unstable.
+- The `readonly` modifier is now available in the reflection API via the `isReadonly()` or `returnsReadonly()` functions.
+- A new option `-vEval.JitAttemptSuperInlining=true` is added to `hhvm` to enable super inlining optimization.
+- The [WriteHandle::truncate] method is added to HSL for writable files.
+- Various APIs in HSL are now considered pure and no longer require the [defaults capabilities](https://docs.hhvm.com/hack/contexts-and-capabilities/introduction).
+- The `--custom-hhi-path` option is added to hh_client and hh_server.
```

Comment:
This should probably be removed, it's primarily for testing experimental features; we can't support people using custom HHIs for real-world situations.